deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -986,8 +986,38 @@ def callback_detect_event(line):
     match = re.match(msg, line)
 
     try:
-        if json.loads(match.group(1))['type'] == 'event':
-            return json.loads(match.group(1))
+        json_event = json.loads(match.group(1))
+        if json_event['type'] == 'event':
+            return json_event
+    except (AttributeError, JSONDecodeError, KeyError):
+        pass
```

Comment:
Shouldn't we add some warning message here? If this happens it may be due to a failure somewhere and we may want to debug it. 

I suggest adding a warning message using `logging.warning`.